### PR TITLE
Upgrade to controller-runtime 0.16.2

### PR DESCRIPTION
**Pull request by roehrich-hpe**

### CODE REVIEW

File: daemons/compute/server/servers/server_default.go [hunk 1]
```diff
@@ -206,11 +208,14 @@ func CreateDefaultServer(opts *ServerOptions) (*defaultServer, error) {
...
+
 	mgr, err := ctrl.NewManager(s.config, ctrl.Options{
-		Scheme:             scheme,
-		LeaderElection:     false,
-		MetricsBindAddress: "0",
-		Namespace:          s.namespace,
+		Scheme:         scheme,
+		LeaderElection: false,
+		Metrics:        metricsserver.Options{BindAddress: "0"},
+		Cache:          cache.Options{DefaultNamespaces: namespaceCache},
```
Comment (bdevcich) [inline daemons/compute/server/servers/server_default.go:218]: I ended up doing this for remora:

```
cache.Options{DefaultNamespaces: map[string]cache.Config{
    s.namespace: {},
}}
```

Then you could get rid of the lines above.

File: go.mod [hunk 2]
```diff
@@ -2,23 +2,31 @@ module github.com/NearNodeFlash/nnf-dm
...
 go 1.19
 
+replace github.com/NearNodeFlash/lustre-fs-operator => ../lustre-fs-operator
+
+replace github.com/NearNodeFlash/nnf-sos => ../nnf-sos
+
+replace github.com/NearNodeFlash/nnf-ec => ../../nnf-ec-1.git
+
+replace github.com/DataWorkflowServices/dws => ../dws
+
```
Comment (bdevcich) [inline go.mod:12]: Reminder

**Review by bdevcich — APPROVED** [review]

File: go.mod (hunk 2 above)
Comment (roehrich-hpe) [reply go.mod]: Fixed

File: daemons/compute/server/servers/server_default.go (hunk 1 above)
Comment (roehrich-hpe) [reply daemons/compute/server/servers/server_default.go]: I'll try that.

**Pull request merged**